cpp/build-support/trim-boost.sh

```diff
@@ -41,6 +41,8 @@ BOOST_LIBS="$BOOST_LIBS regex.hpp"
 BOOST_LIBS="$BOOST_LIBS multiprecision/cpp_int.hpp"
 # These are for Thrift when Thrift_SOURCE=BUNDLED
 BOOST_LIBS="$BOOST_LIBS algorithm/string.hpp locale.hpp noncopyable.hpp numeric/conversion/cast.hpp scope_exit.hpp scoped_array.hpp shared_array.hpp tokenizer.hpp version.hpp"
+#These are for flight
+BOOST_LIBS="$BOOST_LIBS process.hpp process asio fusion"
```

Comment:
I thought you only needed process.hpp? Why also `process asio fusion`?

Reply:
Thanks for your comments. 

Some other header files are referenced (directly or indirectly) from process.hpp. 
I tried to refine sub-directories in process, asio and fusion, but gave up because too many individual individual header files were required.

Reply:
`bcp` should figure that out and pull whatever dependencies are needed into the trimmed boost bundle we build, so if `process.hpp` is all that Flight references, let's just include that.